modules/swagger-codegen/src/main/resources/lagomScalaApi/plugins.sbt.mustache

```diff
@@ -0,0 +1,10 @@
+//
+// Copyright (C) 2016 Lightbend Inc. <https://www.lightbend.com>
```

Comment:
Can we remove it? The autogenerated files are unlicensed by default.

Reply:
Done